### DISCUSSION TITLE
Issues With README Code

### DISCUSSION THREAD

**Discussion by gfleetwood** [Textualize/textual#130] [General]
Hello. I was going to open an issue but I saw this message:

> Textual is currently in active development. I will try to keep the sample functional, but I make no guarantee anything will >run right now.
>
>Unless this is a blatant bug, you might want to open a discussion for now.

I'm using Ubuntu 20.04 and Python 3.7.10 in a conda environment. The first example (and others) in the README doesn't work:

```
from textual.app import App

class Beeper(App):
    def on_key(self):
        self.console.bell()

Beeper.run()
```

It gives this error:

```
TypeError: on_key() takes 1 positional argument but 2 were given
```

I happened to run across some other code in the repo that included an `event` parameter so including that fixes the script:

```
from textual.app import App

class Beeper(App):
   # Change on next line
    def on_key(self, event):
        self.console.bell()

Beeper.run()
```

However, when I fix that and try to use the script I get this error after pressing a button:

```
TypeError: object NoneType can't be used in 'await' expression
```

That I'm assuming is coming from within the package itself. I found this SO answer that might be relevant: 

https://stackoverflow.com/a/62792452/12492315

On an unrelated note, are these scripts supposed to be run as `python script.py` or `python -m script.py`. I'm seeing the latter thrown around in discussion in this repo.

**Comment by willmcgugan** [comment]
It’s likely you are running an old version of Textual.

**Comment by gfleetwood** [comment]
Huh. A `pip --upgrade` fixed the issues. Weird since I only remember installing textual within the past week. Thanks.
